libs/sdk-ui-dashboard/src/model/commandHandlers/drill/drillToDashboardHandler.ts

```diff
@@ -3,6 +3,7 @@ import { SagaIterator } from "redux-saga";
 import { call, put, select } from "redux-saga/effects";
```

Comment:
Could you add some UT tests for  [drillToDashboardHandler.ts](https://github.com/gooddata/gooddata-ui-sdk/pull/3546/files#diff-f49ffc49002d9829c6e6dc5e13cc1850250b138faeb5a7b8922302af416e775c) related to your changes ?